src/modules/freia/generators/MultiBladeGenerator.h

```diff
@@ -18,28 +19,35 @@ using VMM3Data = ESSReadout::VMM3Parser::VMM3Data;
 
 class MultiBladeGenerator : public ReadoutGeneratorBase {
 
-public:
-  //MultiBladeGenerator() : ReadoutGeneratorBase(ESSReadout::Parser::DetectorType::FREIA) {}
+ public:
   MultiBladeGenerator();
 
   struct {
-    std::string Detector;
-    // bool Tof{false};
-    // bool Loki{false}; // implies four amplitudes
-    // bool Debug{false};
+    std::string Detector{"Freia"};
+    bool Tof{false};
+
+    // Instruments
+    bool Freia{false};
+    bool Estia{false};
+    bool Amor{false};
+
+    bool Debug{false};
 
     // Masks are used to restrict the generated data
-    int FiberVals{4};
-    int FiberMask{0xff};   // Fibers 0 - 4
+    int NFibers{4};      // Fibers 0 - 4
+    int FiberMask{0xff};   
```

Comment:
Asmentioned earlier the fiber mask should be longer.

We can figure out sensible default values later.